_topic_maps/_topic_map.yml

```diff
@@ -1143,6 +1143,8 @@ Topics:
   Distros: openshift-enterprise,openshift-origin
 - Name: Using PTP hardware
   File: using-ptp
+- Name: Developing PTP or bare-metal events consumer applications
+  File: ptp-cloud-events-consumer-dev-reference
```

Comment:
do you PTP and BMER two different consumer example ?

Reply:
You tell me :sweat_smile: @jzding Does this consumer app overview work for BMER events?

Reply:
I think you should have consumer implementation section separated for  BMER and PTP